### DISCUSSION TITLE
New Ring indoor Cam (2nd Gen)

### DISCUSSION THREAD

**Discussion by lazyboy0172** [tsightler/ring-mqtt#640] [Feature Requests]
I know this was just released today so obviously no rush, but wanted to know if you knew how feasible it was to support the new 2nd gen ring indoor cam? If it was similar to the car cam and likely going to be unsupported for a while, etc. I picked one of these up today, so if there's anything I can do to help test or confirm these suspicions I'd be happy to help! If I find out it's going to be unlikely to be supported for a while I may way my options to return it and pick up another 1st gen version instead. I liked the privacy shield on the 2nd gen since it'll go in our basement where my kid may want privacy in the future, but I also like having everything in Home Assistant so my son may have to deal with it.

**Comment by tsightler** [comment]
What actual problem do you have with it?  Generally, there's no specific code required to just support a new camera model, it would just display as an "Unknown Model", which isn't so bad.  I wouldn't expect it to be anything like the Car Cam as that device is an entirely new device with new API and everything, at least as far as I can tell.

**Reply by lazyboy0172** [reply]
It's not showing up in mqtt at all which made me wonder if it was using a new api or something like the car cam, being a new device and all. I don't keep up enough with ring stuff though so I didn't know if that information was public or needed to be researched, etc. I recently purchased 3 new products - Indoor cam, Indoor cam (2nd gen), and stick up cam battery... the indoor cam and stick up cam battery both came into mqtt immediately after restarting the service, but the indoor cam (2nd gen) hasn't shown up which led me here. The indoor cam (2nd gen) Does appear in my 'regular' Ring integration, with model <unknown>, but that integration isn't very useful.

**Reply by tsightler** [reply]
If it shows up in the regular integration but not in ring-mqtt that is very odd.  What do the logs show during startup?  Are you using a separate account for ring-mqtt from the primary Ring account?

**Reply by lazyboy0172** [reply]
ah man.. it works now, so maybe I'm just a dummy or it was a fluke somewhere. I just factory reset the device and set it up again from scratch, then restarted ring-mqtt with video streaming add-on to catch the initial log on a freshly added device with a new name. But of course now it's showing up properly and working. So, good news I guess?
